### PR TITLE
Nms deployment

### PR DESCRIPTION
**Pull request by koshalt**
Scripts to handle nms deployment

### CODE REVIEW

**Comment by koshalt** [comment]
@larubbio Do you have time to take a peek at this?

File: roles/db/templates/master/activemq.xml.j2 [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+    <property name="url" value="jdbc:mysql://{{ mds_vip }}/activemq?relaxAutoCommit=true"/>
+    <property name="username" value="{{ appdbuser }}"/>
+    <property name="password" value="{{ appdbpassword }}"/>
+    <property name="poolPreparedStatements" value="true"/>
+  </bean>
+
+    <!--
+        The <broker> element is used to configure the ActiveMQ broker.
+    -->
+    <broker xmlns="http://activemq.apache.org/schema/core" brokerName="localhost" dataDirectory="${activemq.data}">
```
Comment (frankhuster) [inline roles/db/templates/master/activemq.xml.j2]: `schedulerSupport="true"` is missing

**Comment by koshalt** [comment]
@tagating What were these scripts based on? Did you follow the install motech guide or something else?

**Comment by manjotnms** [comment]
@koshalt : Deployment scripts are based on NMS-Deployment_Health_Monitoring document added in spec folder

File: roles/db/templates/master/activemq.xml.j2 (hunk 1 above)
Comment (manjotnms) [inline roles/db/templates/master/activemq.xml.j2]: Thanks!!! Updated conf file

**Comment by koshalt** [comment]
@manjotnms What is the version of the component dependencies? For example, what version of activemq, mysql, etc?

**Pull request merged**